arch/x86/kernel/cpu/resctrl/monitor.c

```diff
@@ -25,6 +25,12 @@
 #include "internal.h"
 
 struct rmid_entry {
+	/*
+	 * Some architectures's resctrl_arch_rmid_read() needs the CLOSID value
+	 * in order to access the correct monitor. This field provides the
```

Comment:
at upstream commit these comments are outside the struct.https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=40fc735b78f0c81cea7d1c511cfd83892cb4d679

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev